packages/wasm/wasm.0.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "wasm"
+version: "0.1"
+maintainer: "Jonathan Protzenko <jonathan.protzenko@gmail.com>"
+authors: "Jonathan Protzenko <jonathan.protzenko@gmail.com>"
+homepage: "https://github.com/WebAssembly/spec"
+bug-reports: "https://github.com/WebAssembly/spec/issues"
+license: "https://github.com/WebAssembly/spec/blob/master/LICENSE"
```

Comment:
It looks like the license is `Apache-2.0` from that link. The type of this field is the name of the license (preferably from SPDX) rather than a link to the license.